interface/forms/newpatient/save.php

```diff
@@ -47,8 +47,8 @@
 
 $nexturl = $normalurl;
 
-$provider_id = $userauthorized ? $_SESSION['authUserID'] : 0;
-$provider_id = $provider_id ? $encounter_provider : $provider_id;
+$provider_id = $_SESSION['authUserID'] ? $_SESSION['authUserID'] : 0;
+$provider_id = $encounter_provider ? $encounter_provider : $provider_id;
 
```

Comment:
whoa, cool, since the only way `$encounter_provider` gets loaded is if it is an auth provider in the dropdown...